src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -115,6 +116,8 @@ public function putOffExpiration(Key $key, $ttl)
 
                 $store->putOffExpiration($key, $adjustedTtl);
                 ++$successCount;
+            } catch (NotExpirableStoreException $e) {
+                $this->logger->notice('The store does not support expiration of locks.', array('store' => $store));
```

Comment:
You should either increment `$successCount` or create a new `$ignoreCount` variable in order to define whether or not the strategy is met.